Jordan_Class.py

```diff
@@ -0,0 +1,202 @@
+                                 # if there is no match though, then it will write the initialized answer
+            if userInput == 'q':
+                print(random.choice(self.values_smalltalk[i]))
+                print("---------------------------------")
+                print("Do you want to choose another topic? Pick below or press 'q' to quit for realsies.")
+                print("---------------------------------")
+                break                                
+            resp = resp.format(word) # replace if applicable      
+            print(resp) # print response
+                
+    def caring_Jordan(self):
+        """
+        
+        A function to talk with Jordan about your deepest sorrows
+        and your destroyed soul.
+        
+        """
+        
+        while True:
+            userInput = input(">>>")
+            resp = "I did not understand what you said. Also, I am sensitive to punctuation." # initalize this
+            counter = 0  # initalize this
+            while resp == "I did not understand what you said. Also, I am sensitive to punctuation." and counter < len(self.keys_caring):
+                for i in range(0, len(self.keys_caring)):
+                    match = self.keys_caring[i].match(userInput)
+                    if match:
+                        word = self.keys_caring[i].split(userInput)[1]  # We have to take the first element of this list
+                        resp = random.choice(self.values_caring[i])
+                    counter += 1
+            if userInput == 'q':
+                print(random.choice(self.values_caring[i]))
+                print("---------------------------------")
+                print("Do you want to choose another topic? Pick below or press 'q' to quit for realsies.")
+                print("---------------------------------")
+                break
+            resp = resp.format(word) # replace if applicable                   
+            print(resp)
+            
+    def cursing_Jordan(self):
+        """
+        
+        A function to insult Jordan and get insults back.
+        
+        """
+        while True:
+            userInput = input(">>> ")
+            resp = "I did not understand what you said. Also, I am sensitive to punctuation." # initalize this
+            counter = 0  # initalize this
+            while resp == "I did not understand what you said. Also, I am sensitive to punctuation." and counter < len(self.keys_cursing):
+                for i in range(0, len(self.keys_cursing)):
+                    match = self.keys_cursing[i].match(userInput)
+                    if match:
+                        word = self.keys_caring[i].split(userInput)[1]  # We have to take the first element of this list                        
+                        resp = random.choice(self.values_cursing[i])
+                    counter += 1
```

Comment:
I feel like I'm seeing a lot repetition in this code and that the only major changes between the Jordan states is the list Jordan searches through.

I'm not saying that this is easy (and maybe it's a bad idea so late in the project), but couldn't you just create one function that you feed different lists as input?

'Jordan_topic(topic)' 

with the topic being a list to search through (e.g., cursing). 
If this is too much, you could at least take elements that are 100% the same, turn them into functions (e.g., the "if userInput == 'q': " section) and paste them.

I got the impression that our teachers care a lot about programming style/elegance, so seeing you repeat so many lines of code, makes me a little worried (not saying that I'm any better, I'm a horrible coder)